scripts/populate_ingest/convert/conversion_utils.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+
+ACCESSION_PATTERNS = {
+    "insdc_project_accessions": "^[D|E|S]RP[0-9]+$",
+    "ega_accessions": "EGA[DS][0-9]{11}",
+    "dbgap_accessions": "phs[0-9]{6}(\\.v[0-9])?(\\.p[0-9])?",
+    "geo_series_accessions": "^GSE.*$",
+    "array_express_accessions": "^E-....-.*$",
```

Comment:
This pattern seems too broad.
After the 1st dash only letters are allowed.
After the 2nd one - only numbers.
See https://www.ebi.ac.uk/arrayexpress/help/accession_codes.html